static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -80,6 +106,52 @@ define([
         return filtered.valueOf();
     }
 
+    function isImageLoaded(img) {
+        return img.complete;
+    }
+
+    function onImagesLoaded() {
+        var images = $('.js-article__body img');
+        return new Promise(function (resolve) {
+            var imgLoadedPromises,
+                notLoaded = _.reject(images, isImageLoaded);
+            if (notLoaded.length === 0) {
+                resolve();
+            } else {
+                imgLoadedPromises = _.map(notLoaded, function (img) {
+                    return new Promise(function (imgResolve) {
+                        bean.on(img, 'load', imgResolve);
+                    });
+                });
+                Promise.all(imgLoadedPromises).then(resolve);
+                window.setTimeout(resolve, 10000); // timeout after 10s and resolve
```

Comment:
Ok. Then are commercial aware that by delaying until all images are loaded will push back their overall ads rendered metric?